deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -419,6 +419,28 @@ spec:
             dependencies:
               description: External dependencies needed by the Infinispan cluster
               properties:
+                artifacts:
+                  items:
+                    properties:
+                      hash:
+                        description: Hash sum of the downloaded file to validate
+                        pattern: ^(sha1|sha224|sha256|sha384|sha512|md5):[a-z0-9]+
+                        type: string
+                      type:
+                        description: Type of file (archive) that will be downloaded. If not specified, the file type will be automatically determined by its extension or downloaded without unpacking
+                        enum:
+                          - file
+                          - zip
+                          - tgz
+                        type: string
+                      url:
+                        description: Url from which the file will be downloaded
```

Comment:
```suggestion
                        description: URL where the archive is available for download.
```

Reply:
@oraNod please see my comment above. It's not only archive, can be any file to download. What is the correct message?